src/components/vue/webhook-utility/AllowedUpdates.vue

```diff
@@ -33,14 +33,14 @@ const allowDefault = () => emit('update:modelValue', DEFAULT_UPDATE_TYPES as any
           type="checkbox"
           class="checked:accent-grammy-600"
           :checked="allowedUpdates[updateType]"
-          @click="set(updateType, $event.target.checked)"
+          @click="set(updateType, ($event.target as any).checked)"
```

Comment:
Why the cast?

Reply:
AFAIK TS can't determine the correct type for `$event.target`, it tells me `checked` does not exist inside, and I know it does.
I don't know why this wasn't producing any errors before tho

Reply:
https://stackoverflow.com/a/69400791

Reply:
So `HTMLInputElement` would be preferred

Reply:
It's still a cast, but okay, I can change it

Reply:
```suggestion
          @click="set(updateType, ($event.target as HTMLInputElement).checked)"
```

Reply:
Done :)